.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,24 @@
+name: Fuzzing
+
+on:
+  schedule:
+    - cron: '* 6 * * *' # Every day at 6:00 UTC
+  workflow_dispatch:  # Allows you to run this workflow manually from the Actions tab
+
+jobs:
+  fuzz:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup libfuzzer
+        run: Fuzzing/install_libclang_fuzzer.sh
+      - name: Cleanup any existing fuzz runs
+        run: rm -rf /tmp/fuzzing/artifacts  # N.B. keeps the corpus
+      - name: Fuzz Mach-O Parsing
+        run: bazel run --config=fuzz //Fuzzing:MachOParse_run -- -- -max_len=32768 -runs=1000000
```

Comment:
Could you use `Fuzzing:all` or add a test_suite() rule that we can reference so we don't have to remember to update this workflow?

Reply:
One advantage of leaving this here as a GitHub action is that we can open up issues automatically for crashes using the GitHub API and reference the fuzzing step that produced it.

Reply:
Using `bazel query` now to iterate over all fuzz targets. Eliminates having to update the actions config, but will also let us open issues with the specific fuzz target which failed.